### PR TITLE
Add gatsby-plugin-eslint to eslint recipe so warnings/errors will print to CLI Learned this was necessary from this issue https://github.com/gatsbyjs/gatsby/issues/23666

### PR DESCRIPTION
**Pull request by KyleAMathews**

### CODE REVIEW

File: packages/gatsby-recipes/recipes/eslint.mdx [hunk 1]
```diff
@@ -27,9 +28,19 @@ add additional presets, plugins, and rules.
...
 <NPMPackage name="eslint-config-react-app" dependencyType={"development"} />
+<NPMPackage name="gatsby-plugin-eslint" dependencyType={"development"} />
+
+---
+
+Install `gatsby-plugin-eslint`
+
+<GatsbyPlugin name="gatsby-plugin-eslint" />
 
 ---
```
Comment (pvdz) [inline packages/gatsby-recipes/recipes/eslint.mdx:40]: Typobot is correct...

File: packages/gatsby-recipes/recipes/eslint.mdx [hunk 2]
```diff
@@ -27,9 +28,19 @@ add additional presets, plugins, and rules.
...
 
+Write out a basic .eslintrc.js file that you can extend with your own choosen
```
Comment (pvdz) [inline packages/gatsby-recipes/recipes/eslint.mdx:41]: Typobot is correct...

Comment (herecydev) [inline packages/gatsby-recipes/recipes/eslint.mdx:41]:
```suggestion
Write out a basic .eslintrc.js file that you can extend with your own chosen
```

**Comment by muescha** [comment]
is #23666 is fixed with this PR?

**Comment by herecydev** [comment]
Not really from my perspective. It would still be nice to add a few rules without copy/pasting gatsby's default ruleset.

This is important as those rules are adjusted over time.

**Pull request merged**